{"uid":"abe6caae1f3c11fb","name":"Intentar convertir una cantidad negativa","fullName":"Gestión de monedas: Intentar convertir una cantidad negativa","historyId":"8d414140179c00af36a2d4d15ce13002","time":{"start":1732867109389,"stop":1732867109395,"duration":6},"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'error_message'\n","statusTrace":"  File \"C:\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\currency_manager_steps.py\", line 68, in step_impl\n    assert context.error_message == \"La cantidad debe ser positiva.\"  # Verificamos el error de cantidad negativa\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":9,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'error_message'\n","statusTrace":"  File \"C:\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\currency_manager_steps.py\", line 68, in step_impl\n    assert context.error_message == \"La cantidad debe ser positiva.\"  # Verificamos el error de cantidad negativa\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","steps":[{"name":"Given un usuario intenta convertir -50 USD a EUR","time":{"start":1732867109395,"stop":1732867109395,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When realiza la conversión","time":{"start":1732867109395,"stop":1732867109395,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then se muestra un mensaje indicando que la cantidad debe ser positiva","time":{"start":1732867109395,"stop":1732867109395,"duration":0},"status":"broken","statusMessage":"AttributeError: 'Context' object has no attribute 'error_message'\n","statusTrace":"  File \"C:\\Python312\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps\\currency_manager_steps.py\", line 68, in step_impl\n    assert context.error_message == \"La cantidad debe ser positiva.\"  # Verificamos el error de cantidad negativa\n           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":3,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Gestión de monedas"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"4adb3f7ca6f8f55a","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732866875843,"stop":1732866875848,"duration":5}},{"uid":"32b86d4c162ce201","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732866758672,"stop":1732866758678,"duration":6}},{"uid":"2706dc301cb4aeee","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732866673033,"stop":1732866673036,"duration":3}},{"uid":"8ecc079dd27eb849","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732866135764,"stop":1732866135768,"duration":4}},{"uid":"3fdcb59dc32276de","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732865902865,"stop":1732865902870,"duration":5}},{"uid":"2700af11a96f1b43","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732864772865,"stop":1732864772870,"duration":5}},{"uid":"75362a89c11b0b4","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732864689124,"stop":1732864689127,"duration":3}},{"uid":"108ea876ba274129","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732863642166,"stop":1732863642170,"duration":4}},{"uid":"fd20c4499b80f1d7","status":"broken","statusDetails":"AttributeError: 'Context' object has no attribute 'error_message'\n","time":{"start":1732862762799,"stop":1732862762802,"duration":3}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"abe6caae1f3c11fb.json","parameterValues":[]}